src/Nexcess/GithubWebhookJira/Webhook.php

```diff
@@ -23,11 +23,11 @@
  * IN THE SOFTWARE.
  */
 
-namespace GithubWebhookJira;
+namespace Nexcess\GithubWebhookJira;
```

Comment:
Sorry, change of plans. Decision has been made to keep `NocWorx` as the namespace for software projects.